### ISSUE TITLE
Update latest Atom docset

### ISSUE THREAD

**Issue by ooJerryLeeoo** [Kapeli/Dash-User-Contributions#1406]
as the title.

**Comment by Kapeli** [comment]
@Ephemera and @bkbooth can you take a look at this when you can please?

**Comment by bkbooth** [comment]
It's been ages since I've looked at this or the Atom docs, but I'll try and carve out some time to update it (unless @Ephemera has time before I do). At least the Atom API is at 1.x now, should be more stable.

**Comment by bkbooth** [comment]
I spent a little bit of time on this to try and gauge how much work to get my Atom docs scraper working again, it's a fair bit, the structure of the docs have changed almost completely since I wrote my scraper. I'll try and get it done in the next few days when I can get some time.

**Comment by ooJerryLeeoo** [comment]
@bkbooth Any update on this ?

**Comment by bkbooth** [comment]
In short, there's not too much to update other than that I have progressed with it.

In a few more words, I'm basically rewriting my docs scraper from scratch because (as I mentioned previously) the Atom docs structure has changed almost completely since I wrote my original scraper (Atom v0.120) and my old method had a bunch of manual steps to manipulate the HTML. I'm trying to automate the scraping and manipulation of the HTML this time, so whenever Atom has a new release I can just run the script with a new API version number. I've basically solved all of the more complicated issues that I wasn't totally sure about with the HTML processing so I'm hoping to finish this when I next get a chance to work on it (probably tonight GMT+10). The next step is to process the HTML files to pull out the classes, methods, properties, etc. to populate the Dash docset database. I haven't made a start on this step yet, but having done it previously I'm reasonably confident that I shouldn't run into any major issues with it. None of this is particularly time consuming, it's only slow because I don't actually have that much spare time to consume at the moment.
